service/src/main/kotlin/fi/espoo/evaka/invoicing/service/IncomeCoefficientMultiplierProvider.kt

```diff
@@ -0,0 +1,26 @@
+// SPDX-FileCopyrightText: 2017-2023 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.invoicing.service
+
+import fi.espoo.evaka.invoicing.domain.IncomeCoefficient
+import java.math.BigDecimal
+
+fun interface IncomeCoefficientMultiplierProvider {
+    fun multiplier(coefficient: IncomeCoefficient): BigDecimal
+}
+
+class EspooIncomeCoefficientMultiplierProvider : IncomeCoefficientMultiplierProvider {
```

Comment:
It would be more consistent to move this class under `src/main/kotlin/fi/espoo/evaka/espoo`, because we're slowly migrating all espoo-specific code and configuration there anyway.

Reply:
Makes sense, I will move it there.